packages/@ember/-internals/glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2904,6 +2904,84 @@ moduleFor(
       this.assertText('initial value');
     }
 
+    ['@test GH#18147 - a two way binding flows upstream to a parent component through a CP']() {
```

Comment:
Typo in the issue ID - should be 18417 instead of 18147.

Reply:
Fixed